assets/src/web-stories-block/block/icons.js

```diff
@@ -17,7 +17,7 @@
 /**
  * WordPress dependencies
  */
-import { Path, Rect, SVG } from '@wordpress/components';
+import { Circle, Path, Rect, SVG } from '@wordpress/components';
```

Comment:
The icons in this file are the ones we can't just put into `.svg` files, right? Because of the block toolbar?

Reply:
the icons and some copy updates are done in this approved [PR](https://github.com/rtCamp/web-stories-wp/pull/208/files#diff-aaed123f303b32e5aaa1f595259a98e27016761e29d02e395b44aa8c5b4839e1R1). we have moved required icons to svg.